### ISSUE TITLE
UX: Publish a flow definition to the registry

### ISSUE THREAD

**Issue by aperepel** [Chaffelson/nipyapi#32]
As a deployer, I want to publish a given flow version to the registry. The command-line expectation is to provide a local file (or, better, URI, where file://whatever is just one instance) to be published. Additionally, one would provide mnemonic flow path in the registry and the system must resolve the correct bucket/item uuids.

Bonus: support 'create parent' buckets/items if not available via a simple cmd-line switch. E.g. think of the `mkdir -p /path/which/may/not/yet/exist`

**Comment by Chaffelson** [comment]
Dependent on #30

**Comment by Chaffelson** [comment]
The import code for this is now in master @aperepel 
I have not included parent creation or mnemonics, opting to get the import/export base functions out the door now, and look into convenience wrappers later.

**Comment by Chaffelson** [comment]
Closing this as the core of the solution is ready for release - will create a separate issue to allow import from URI/FILE/WEB/etc.
